README.md

```diff
@@ -1 +1,85 @@
 # Ruby Sierra API Client
+
+Simple client for querying the Sierra API
+
+## Version
+
+> 1.0.0
+
+## Using
+
+1. Configure the client with Sierra API credentials
+2. Make requests
+
+### Configuration
+
+Example configuration:
+
+```ruby
+require 'nypl_sierra_api_client'
+
+client = SierraApiClient.new({
+  base_url: "https://[fqdn]/iii/sierra-api/v5", # Defaults to ENV['SIERRA_API_BASE_URL']
+  client_id: "client-id", # Defaults to ENV['SIERRA_OAUTH_ID']
+  client_secret: "client-secret", # Defaults to ENV['SIERRA_OAUTH_SECRET']
+  oauth_url: "https://[fqdn]/iii/sierra-api/v3/token", # Defaults to ENV['SIERRA_OAUTH_URL'],
+  log_level: "debug" # Defaults to 'info'
+})
+```
+
+### Requests
+
+Example GET:
+
+```ruby
+bib_response = sierra_client.get 'patrons/1234'
+bib = bib_response.body
```

Comment:
I have not worked with the Sierra API yet. Does the patrons endpoint return data related to bibs?

Reply:
That's a typo. Good catch. It's now ` bib_response = sierra_client.get 'bibs/1234' `